src/vs/workbench/contrib/chat/common/chatSessionsService.ts

```diff
@@ -111,6 +112,11 @@ export interface IChatSession extends IDisposable {
 	readonly isCompleteObs?: IObservable<boolean>;
 	readonly interruptActiveResponseCallback?: () => Promise<boolean>;
 
+	/**
+	 * Editing session transferred from a previously-untitled chat session in `onDidCommitChatSessionItem`.
+	 */
+	initialEditingSession?: IChatEditingSession;
```

Comment:
This is not the nicest thing, but it does the job, let me know if you have a preference for something better